### PR TITLE
Portfolio - Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by aamirkhan2478**
### My Portfolio - Mobile Version

> This is my portfolio website only for mobile-only now. In the future, I will complete this website and use this site for my portfolio.

### Built with

- HTML
- CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <body>
+    <header>
+      <nav class="navbar">
+        <ul class="nav-flex">
+          <li class="nav-link"><a href="#">John Doe</a></li>
+          <li class="nav-link">
+            <a href="#"><i class="fa-solid fa-bars"></i></a>
+          </li>
+        </ul>
+      </nav>
```
Comment (ShahierNashaat) [inline index.html:27]: - [ ] Kindly be noticed that you should end the `header` tag after the `nav` tag ie.
```
<header>
      <nav class="navbar">
        <ul class="nav-flex">
          <li class="nav-link"><a href="#">John Doe</a></li>
          <li class="nav-link">
            <a href="#"><i class="fa-solid fa-bars"></i></a>
          </li>
        </ul>
      </nav>
</header>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hesitate to contact me.
+        </p>
+        <div class="social-media-icons">
+          <a href="#"><i class="fa-brands fa-github"></i></a>
+          <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+          <a href="#"><i class="fa-brands fa-angellist"></i></a>
+          <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          <a href="#"><i class="fa-brands fa-medium"></i></a>
+        </div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        <img
+          class="main-image"
+          src="/assets/images/main-image.svg"
+          alt="main-image"
+        />
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly be noticed that the decorations are always background images, not `img` tags.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @aamirkhan2478 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @aamirkhan2478        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**